### ISSUE TITLE
Accepting URL inputs

### ISSUE THREAD

**Issue by asiripanich** [AURIN-OFFICE/atrc-workflow-runner#6]
@asiripanich made a comment about the workflow runner accepting URL inputs which I think has disappeared - +1 for that from me.

_Originally posted by @trisreed in https://github.com/AURIN-OFFICE/atrc-workflow-runner/issues/5#issuecomment-1077280186_

**Comment by javeddc** [comment]
@trisreed @asiripanich My angle on this is we've been trying to avoid external calls to the wider internet being made as part of workflow execution (wether from containers themselves or from the workflow runner) for simplicity and reproducibility. By limiting inputs to things we have control of (AURIN catalogue data) or keep in full (tool output, user uploaded input), we can re run (and potentially cache) things with a certainty that they are the same. There's also a bit of security to deal with when allowing requests. 

Open to hearing what the value is in having this though. 

But IIUC, it's not a requirement for this case study, right?

**Comment by trisreed** [comment]
@javeddc good point, perhaps there could be a way to create tool/service that fetches an external resource ;) certainly not part of the SDCS.

**Comment by asiripanich** [comment]
Thanks for a clear explanation. ;)
